src/pymodaq/utils/managers/parameter_manager.py

```diff
@@ -29,22 +26,30 @@ def __init__(self):
 
         self.tree.setMinimumWidth(150)
         self.tree.setMinimumHeight(300)
+        
+        self.setup_actions(action_list) # Making the button
+        if len(action_list) != 0: # If no action allowed, no need for splitter
```

Comment:
I don't like having a widget in some cases and not in other cases because if one calls it from an external program it could lead to an error... Its no big deal to leave the splitter especially because it is closed by default...